### PR TITLE
BD103's Page (WIP)

### PR DESCRIPTION
**Pull request by BD103**
Don't merge yet, uwu.

(This is my page on the website!)

### CODE REVIEW

**Comment by whippingdot** [comment]
Is this done? Should I review the changes yet or...

File: views/BD103.html [hunk 1]
```diff
@@ -4,13 +4,39 @@
...
+      background-repeat: no-repeat;
+      background-size: cover;
+      background-attachment: fixed;
+    }
+    h1, h2 {font-family: "Pangolin", cursive;}
+    .title {font-size: 60px;}
+    .center {text-align: center;}
+    .left {text-align: left;}
+    .right {text-align: right;}
+  </style>
```
Comment (darkdarcool) [inline views/BD103.html:23]: Please make this a but tidier by adding two indentations for each css element like this:

``` css
body {
        font-family: "Montserrat", sans-serif;
        font-size: 20px;
        background: linear-gradient(to bottom right, darkcyan, lightcyan);
        background-repeat: no-repeat;
        background-size: cover;
        background-attachment: fixed;
 }
    h1, h2 {font-family: "Pangolin", cursive;}
        .title {font-size: 60px;}
        .center {text-align: center;}
        .left {text-align: left;}
        .right {text-align: right;}
  }
```
Please update that.

**Review by darkdarcool — CHANGES_REQUESTED** [review]
Looks good to me! You might wanna make the css a bit more tidy, like making two indentations for each changed thing, l made comment on what to change.

File: views/BD103.html (hunk 1 above)
Comment (whippingdot) [inline views/BD103.html:23]: Umm sorry, I also have a problem with this. I already said that you should put your CSS in the style.css file(I even deleted everyone's files earlier cause they didn't do that). It is tidier that way. What I told everyone is to put a class on their HTML tag, ex. `<HTML class="bd">`, and then in the style.css file they can create a section for themselves, with comments, and for the CSS properties we could have:
```html
.bd body {
      font-family: "Montserrat", sans-serif;
      font-size: 20px;
      background: linear-gradient(to bottom right, darkcyan, lightcyan);
      background-repeat: no-repeat;
      background-size: cover;
      background-attachment: fixed;
 }
.bd h1, h2 {font-family: "Pangolin", cursive;}
.bd .title {font-size: 60px;}
.bd .center {text-align: center;}
.bd .left {text-align: left;}
.bd .right {text-align: right;}
```
And you could add the @import font thing at the top of the file with the other ones.

File: views/BD103.html [hunk 2]
```diff
@@ -4,13 +4,39 @@
 <head>
```
Comment (whippingdot) [inline views/BD103.html:4]: Above this where the HTML tag is put the `<html class="bd">`

File: views/BD103.html [hunk 3]
```diff
@@ -4,13 +4,39 @@
...
+  <h2 class="left">Find me on...</h2>
+  <ol class="right">
+    <li><a href="//repl.it/@BD103">Repl.it</a></li>
+    <li><a href="//github.com/BD103">Github</a></li>
+    <li><a href="//pypi.org/user/BD103">PyPI</a></li>
+    <li><a href="//bd103.github.io">Website</a></li>
+  </ol>
+
+  <h2 class="left">About</h2>
+  <p class="right">I started my programming career in 2017.</p>
```
Comment (whippingdot) [inline views/BD103.html:39]: Thanks for ending the redirect, it was horrible. The content of the page is fine for now I think, but maybe add more to the p element like what languages you know, and stuff like that. Also you can just link accounts from stuff like dev.to too. Also, if you are not redirecting, maybe making it open in a new page **_should_** be removed if not done already.

**Review by whippingdot — CHANGES_REQUESTED** [review]
Added many reviewing comments and gave suggestions. You can change the CSS file(tho be careful as there will be merge conflicts) and you can change the index and signup pages(if the new tab thing is still there).

**Comment by BD103** [comment]
Bruh I'm not done yet.

**Comment by BD103** [comment]
Please wait for review until this PR does not have the WIP title or is not a draft.

**Comment by whippingdot** [comment]
Hey @BD103 I don't wanna be impatient but what is the status on this? Doesn't seem like you have made any changes since I reviewed last time. If you meant to commit, please do.

**Comment by BD103** [comment]
Oooo yeah, got it.

**Comment by whippingdot** [comment]
hey this is the only thing left

**Review by darkdarcool — CHANGES_REQUESTED** [review]
Maybe add a bit more about yourself as a coder.

**Comment by BD103** [comment]
> Maybe add a bit more about yourself as a coder.

Still a draft, please just give me time.

**Comment by darkdarcool** [comment]
Ok. I just left my review when requested.

**Comment by whippingdot** [comment]
@BD103 sry do it asap. Ima share now if i have time but if not till tomorrow

**Comment by darkdarcool** [comment]
@BD103 kk. Just add it to the repl. Ima close this pull for now tho.

**Pull request closed without merging**